app-nuxtjs-3-starter/README.md

```diff
@@ -16,27 +16,76 @@ npm install
 # pnpm
 pnpm install
 ```
+### Environment Variables
 
-## Development Server
+First, copy the `.env.example` file and rename to `.env`.
 
-Start the development server on `http://localhost:3000`
+
+### Secure Tunnel
+
+1. Create a tunnel for you application using [ngrok](https://ngrok.com/). To create a specific domain tunnel as listed below, you need to set up an account.
+
+2. Adjust `BASE_URL`
+   inside the `.env` file to your tunnel domain (e.g.: https://<YOUR-HOST-NAME>.ngrok.io).
```

Comment:
```suggestion
   inside the `.env` file to your tunnel URL, using HTTPS. For example, https://myngroksubdomain.ngrok.io.
```

ngrok can generate subdomains with `.eu`, which is easy missed. So it might be better to substitute the whole URL here.

Also important to use HTTPS.